avocado/core/html.py

```diff
@@ -130,19 +130,20 @@ def tests(self):
                    "INTERRUPTED": "danger"}
         test_info = []
         results_dir = self.results_dir(False)
-        for t in self.result.tests:
-            formatted = {}
-            logdir = os.path.join(results_dir, 'test-results', t['logdir'])
-            formatted['logdir'] = os.path.relpath(logdir, self.html_output_dir)
+        for tst in self.result.tests:
+            tst = tst.copy()    # we don't want to override other's results
```

Comment:
I wanted to use `copy.deepcopy`, but when I looked at it it looked safe to use just a plain `copy`. Did you found those issues, should I change it to the `deepcopy` again?